Package.swift

```diff
@@ -27,6 +27,10 @@ var additionalCompilerFlags: [PackageDescription.SwiftSetting] = {
         ?? []
 }()
 
+var ciCompilerFlags: [PackageDescription.SwiftSetting] = [
+    // REPLACE_WITH_DEFINES_HERE
+]
```

Comment:
Xcode Cloud hates the `Local.xcconfig` and the RevenueCat and RevenueCatUI frameworks don't see the file correctly. So this is a "hardcoded" way to add the `PAYWALL_COMPONENTS` flag in the preprocessor CI script 🤷‍♂️